docs/_src/usage/usage/domain_adaptation.md

```diff
@@ -69,7 +69,73 @@ Annotate also supports two different workflows:
 
 ## User Feedback
 
-A simpler and faster process to retrain models is to utilise user feedback.
-Users can give thumbs up or thumbs down to search results through the Rest API
-and these labels will be stored in the `DocumentStore`
-where they can already be used to retrain the `Reader`.
+A simpler and faster process to retrain models is to utilise user feedback. 
```

Comment:
"... process to fine-tune models to your domain"? 
Retraining sounds to me rather like "starting from a fresh model instance with random weights"

Reply:
Agreed